projects/storefrontlib/src/cms-components/configurator/commons/config-group-menu/config-group-menu.component.html

```diff
@@ -1,41 +1,51 @@
-<nav>
-  <ul class="cx-config-menu" tabindex="0">
-    <ng-container *ngIf="displayedParentGroup$ | async as parentGroup">
-      <li
-        tabindex="0"
-        (click)="navigateUp()"
-        (keyup)="navigateUpOnEnter($event)"
-        class="cx-config-menu-back"
-        *ngIf="parentGroup !== null"
-      >
-        <!--TODO: "Back" text is still under discussion -->
-        <a role="link"
-          ><cx-icon [type]="iconTypes.CARET_LEFT"></cx-icon
-          >{{ 'configurator.button.back' | cxTranslate }}</a
+<ng-container *ngIf="configuration$ | async as configuration">
+  <nav>
+    <ul class="cx-config-menu" tabindex="0">
+      <ng-container *ngIf="displayedParentGroup$ | async as parentGroup">
+        <li
+          tabindex="0"
+          (click)="navigateUp()"
+          (keyup)="navigateUpOnEnter($event)"
+          class="cx-config-menu-back"
+          *ngIf="parentGroup !== null"
         >
-      </li>
-    </ng-container>
-
-    <ng-container *ngIf="displayedGroups$ | async as groups">
-      <ng-container *ngIf="currentGroup$ | async as currentGroup">
-        <ng-container *ngFor="let group of groups">
-          <li
-            (click)="click(group)"
-            (keyup)="clickOnEnter($event, group)"
-            class="cx-config-menu-item"
-            tabindex="0"
+          <!--TODO: "Back" text is still under discussion -->
+          <a role="link"
+            ><cx-icon [type]="iconTypes.CARET_LEFT"></cx-icon
+            >{{ 'configurator.button.back' | cxTranslate }}</a
           >
-            <a [class.active]="currentGroup.id === group.id" role="link"
-              >{{ group.description }}
+        </li>
+      </ng-container>
 
-              <cx-icon
-                *ngIf="configuratorGroupsService.hasSubGroups(group)"
-                [type]="iconTypes.CARET_RIGHT"
-              ></cx-icon
-            ></a>
-          </li>
+      <ng-container *ngIf="displayedGroups$ | async as groups">
+        <ng-container *ngIf="currentGroup$ | async as currentGroup">
+          <ng-container *ngFor="let group of groups">
+            <li
+              (click)="click(group)"
+              (keyup)="clickOnEnter($event, group)"
+              class="cx-config-menu-item {{
+                getGroupStatus(group.id, configuration) | async
+              }}"
+              tabindex="0"
+            >
+              <a [class.active]="currentGroup.id === group.id" role="link"
+                >{{ group.description }}
+                <div class="groupIndicator">
```

Comment:
There is not any condition that proves which group indicator should be displayed. Does it mean that both group indicators are present in the DOM tree?

Reply:
Yes, both will be present and displayed via CSS.
Otherwise we would need to call the getGroupStatus method several Times